docker/gpu/tensorrtdeepsort/tensorrtdeepsort.json

```diff
@@ -0,0 +1,73 @@
+{   
```

Comment:
are we sure that we want to make this .json public?
I did the same mistake by commiting this one: https://github.com/nikostsagk/rasberry_perception/blob/tensorrt_docker_v2/docker/gpu/detectron2/mask_rcnn_R_50_FPN_3x.yaml and this one:
https://github.com/nikostsagk/rasberry_perception/blob/tensorrt_docker_v2/docker/jetson/detectron2/mask_rcnn_R_50_FPN_3x.yaml

and it will be hard to clean it from git history